bindings/python/pyproject.toml

```diff
@@ -1,5 +1,6 @@
 [build-system]
 requires = ["setuptools", "wheel", "setuptools-rust"]
+build-backend = "setuptools.build_meta"
```

Comment:
@Narsil @n1t0 - I had to add this `build-system` dependency to make `pip install -e .` work for me (which in turn I needed to make `make style` work...).

If I don't add this `pip install -e .` fails with this error: https://github.com/pypa/setuptools/issues/1694

Reply:
Awesome thanks! I'm used to directly use `python setup.py install|develop` so I didn't even know about this problem.